testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1332,7 +1332,7 @@ def server_secret
 end
 
 Then(/^port "([^"]*)" should be (open|closed)$/) do |port, selection|
-  output, _code = $server.run("lsof -i -P -n  | grep #{port}  || echo 'closed'", check_errors: false)
+  output, _code = $server.run("ss --listening --numeric | grep #{port}  || echo 'closed'", check_errors: false, verbose: true)
```

Comment:
That should work fine. Filtering the entry in options is probably even more reliable.

```suggestion
  output, _code = $server.run("ss --listening --numeric -H src :#{port}  || echo 'closed'", check_errors: false, verbose: true)
```

Reply:
As you mentioned in Slack, we need some command after `ss` since it always returns `0` as [return value](https://unix.stackexchange.com/questions/384920/ss-exit-code-always-0).
I will add a grep after the command again.